scripts/bash/getDeployablePackages.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash -e
 
-echo "admin-panel" "lesmis" "psss" "web-frontend" "admin-panel-server" "entity-server" "lesmis-server" "meditrak-server" "psss-server" "report-server" "web-config-server"
+echo "admin-panel" "lesmis" "psss" "web-frontend" "report-server" "admin-panel-server" "entity-server" "lesmis-server" "meditrak-server" "psss-server" "web-config-server"
```

Comment:
Turns out this is also an issue. Now `admin-panel-server` depends on `report-server` so they need to be built in the correct order. I think when we get `lerna` these kinds of issues will resolve themselves, right @IgorNadj ?

Reply:
Yeah there's a few circular dependancies at the moment which are untangled in the followup PR.